### DISCUSSION TITLE
Can reacnetgeneror support other elements such as N,Al,B,Mg?

### DISCUSSION THREAD

**Discussion by njustcodingjs** [deepmodeling/reacnetgenerator#877] [Q&A]
Dear developors of reacnetgenerator:
  I am a PhD student studying about energetic materilals. My system contains N and Al elements. I test your program on my systems, but failed. Can reacnetgeneror support other elements such as N,Al,B,Mg?
 Thanks!

**Comment by njzjz** [comment]
It should be OK. What does "failed" mean?
